Templates/index.md

```diff
@@ -1,32 +1,36 @@
 # How to Use EOS Doc Templates
 
-To submit your documentation to the EOS Network Foundation (ENF), create a fork of the receiving repository and then add work on your own fork. From there, do a PR from your fork to the main branch. DevRel then reviews the PR, suggests changes, if any, and after approval the PR can be merged. 
+The DOS Doc Templates are a guide for contributors to submit content to the EOS Documentation Portal. 
 
-ENF Templates use MarkDown language.
+The Templates provide the following:
 
-Templates provide a structure for your content.
+* Provide a structure and guidelines for your content
 
-Text identified within brackets [...] are instructions to you for the type of information to include in the topic.
+* Use MarkDown language.
 
+* Have instructions for the type of information for your content within brackets [...]
+
+* Optional section titles. 
 If a section title contains "(Optional)" and you don’t need to add information for that topic, add the description “N/A”.
```

Comment:
This will render on the previous line (next to "Optional section titles"), so better to move it next to it to avoid confusion.

Reply:
Done